### PR TITLE
Include coin type in backups and fix nodejs binding

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Include coin type in backups and fix nodejs binding

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Example

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/account/builder.rs [hunk 1]
```diff
@@ -47,7 +45,7 @@ impl AccountBuilder {
     pub fn new(
         accounts: Arc<RwLock<Vec<AccountHandle>>>,
         client_options: Arc<RwLock<ClientOptions>>,
-        coin_type: u32,
+        coin_type: Arc<Mutex<u32>>,
```
Comment (thibault-martinez) [inline src/account/builder.rs]: Does it really need to be Arc Mutex already in the parameters ?

Comment (Thoralf-M) [reply src/account/builder.rs]: It's annoying, it's needed unless we do the new method async or change something else I don't know where

Comment (pvdrz) [reply src/account/builder.rs]: Can't it be `Arc<AtomicU32>`?

Comment (Thoralf-M) [reply src/account/builder.rs]: Probably, forgot about that one

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**